routes/misc.php

```diff
@@ -22,3 +22,5 @@
 $this->get('sitemap/{sitemap?}', ['uses' => 'SitemapController@index', 'as' => 'sitemap']);
 
 $this->get('Search', ['uses' => 'SearchController@index', 'as' => 'search']);
+
+$this->get('token', ['uses' => 'User\SessionTokenController@generateToken']);
```

Comment:
A tu powrót do korzeni - skoro wszystko zostało przeniesione do middleware `api`, to `api` jest z definicji bezstanowe. A w takim razie znowu wraca potrzeba generowania tokenów powiązanych z sesją - czyli musi to robić strona która parsuje ciasteczka, czyli 'web'.